preppal-be/models/user.ts

```diff
@@ -0,0 +1,41 @@
+import mongoose from "mongoose";
+const Schema = mongoose.Schema;
+
+const UserSchema = new Schema({
+    userName: {
+        type: String,
+        required: true
+    },
+    password: {
+        type: String,
+        required: true
+    },
+    firstName: {
```

Comment:
Let's delete these. @Rloeppky doesn't ask for first or last name in the registration

Reply:
removed